salt/cluster_node/ha/iscsi_initiator.sls

```
@@ -10,6 +10,15 @@ lsscsi:
       attempts: 3
       interval: 15
 
+{% if grains['osrelease_info'][0] == 15 and grains['osrelease_info'][1] == 2 %}
```

what happens with future versions?

---

> @arbulu89 I think we should target this change to master and then forward-port it to develop.

The things is that it wouldn't be a direct port. The applied change is even in different files, so the commit must be different I guess.

---

> what happens with future versions?

Frankly, I don't even know what will happen in older versions... What I know now is that the new version only has been shipped in SLE15SP2, and not in the other versions.

---

an alternative approach would be

```
start_iscsi_init:
  module.run:
    - service.start:
      - name: iscsi-i
    - - check_cmd:
         '/usr/bin/true` (which always be true)

```
It might seems a bit ugly but on the other side we spare the `15sp3` and later checks maintenance.
```

---

@MalloZup I don't understand what that command would help on. If it's always true, what is the benefit of using it?

---

the benefits is that it will run always and execute, nevertheless the distro. 

it is like `mkdir -p`            no error if existing, make parent directories as needed

---

@MalloZup as you said, it's a bit ugly: it would hide actual failures of the service, and our future selves would have a harder time understanding what's that for.